src/helpers/apifyHelper.js

```diff
@@ -11,12 +9,12 @@ const DEFAULT_POOLING_INTERVAL = 2000; // ms
  * await sleepPromised(3000);
  * @param ms
  */
-export const sleepPromised = ms => new Promise(resolve => setTimeout(resolve, ms));
+const sleepPromised = ms => new Promise(resolve => setTimeout(resolve, ms));
 
 /**
  * Asynchronously waits until execution is finished
  */
-export async function waitUntilFinished(executionId, crawlerClient, interval = DEFAULT_POOLING_INTERVAL) {
+async function waitUntilFinished(executionId, crawlerClient, interval = DEFAULT_POOLING_INTERVAL) {
     let running = true;
 
     while (running) {
```

Comment:
You could use `wait` parameter to make it more efficient

Reply:
It makes just one request every 2mins. But you would have to make multiple requests anyway to support longer run, so lets leave it as it is.